src/npc/npc.gd

```diff
@@ -0,0 +1,31 @@
+extends Area2D
+
+@export var dialogue_array: Array[String] = []
+@export var is_cow: bool = true
+
+@export var dialog_resource: DialogueResource
+@export var dialog_start: String = "start"
+
+# Should this go into a singleton?
+const dialogBox = preload("res://assets/dialogue_boxes/balloon.tscn")
+
+
+# Called when the node enters the scene tree for the first time.
+func _ready() -> void:
+	add_to_group("npc")
+	if !is_cow:
+		self.get_node("Sprite2D").region_rect = Rect2(288,144,16,16)
```

Comment:
if we can't set the region of an exported texture then we should cut the 16x16 sprites we need from the texture itself and just add those